api/personal.example.gradle

```diff
@@ -0,0 +1,6 @@
+// Also log stdout and stderr when running tests
```

Comment:
So is the idea that I copy this to personal.gradle if I want stdout and stderr?

It might be easier to just have the Gradle target have the relevant lines commented out with a comment saying you can uncomment them to get more output; or add a flag you can pass in to enable it.

Reply:
It is easier, but then you have an annoying item in your git status. I like the idea of a flag. I'll see if I can figure that out. If you already know, please do share.